### PR TITLE
Bump conda versions to v1.0.0

### PR DESCRIPTION
**Pull request by jetuk**

### CODE REVIEW

**Comment by jetuk** [comment]
I'm confused by appveyor. Why is it still running 2.7? It doesn't seem to be using the update yml file?

**Comment by jetuk** [comment]
@snorfalorpagus any ideas what I've done wrong?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -65,6 +66,7 @@ requirements:
     - packaging
     - jupyter
     - matplotlib
+    - setuptools_scm
```
Comment (snorfalorpagus) [inline recipe/meta.yaml]: Is this really a runtime dependency? I thought it was just used in setup.py.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% set name = "pywr" %}
-{% set version = "0.5" %}
-{% set git_rev = "v0.5" %}
+{% set version = "1.0.0" %}
+{% set git_rev = "v1.0.0" %}
```
Comment (snorfalorpagus) [inline recipe/meta.yaml:3]: Does setuptools_scm work when the tags have a "v" prefix?

File: recipe/meta.yaml (hunk 1 above)
Comment (jetuk) [reply recipe/meta.yaml]: Yeah maybe not.

File: recipe/meta.yaml (hunk 2 above)
Comment (jetuk) [reply recipe/meta.yaml:3]: It seems to. Checkout the pywr master and it works.

**Comment by jetuk** [comment]
@snorfalorpagus any chance you can move the appveyor integration to the pywr organisation? I can't see what's going on in appveyor atm.

**Comment by jetuk** [comment]
Can we merge this now?

**Pull request merged**